go.mod

```diff
@@ -1,5 +1,40 @@
 module github.com/openshift/must-gather
 
-go 1.17
+go 1.19
 
-require github.com/openshift/build-machinery-go v0.0.0-20210423112049-9415d7ebd33e
+require (
+	github.com/jaypipes/ghw v0.11.0
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.27.4
+	github.com/openshift-kni/debug-tools v0.1.10
+	github.com/openshift/build-machinery-go v0.0.0-20230306181456-d321ffa04533
+	github.com/spf13/cobra v1.6.0
+)
+
+require (
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
+	github.com/go-logr/logr v1.2.3 // indirect
+	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/golang/protobuf v1.5.3 // indirect
+	github.com/google/cadvisor v0.43.0 // indirect
+	github.com/google/go-cmp v0.5.9 // indirect
+	github.com/google/uuid v1.1.2 // indirect
+	github.com/inconshreveable/mousetrap v1.0.1 // indirect
+	github.com/mistifyio/go-zfs v2.1.2-0.20190413222219-f784269be439+incompatible // indirect
+	github.com/moby/sys/mountinfo v0.4.1 // indirect
+	github.com/nxadm/tail v1.4.8 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	golang.org/x/net v0.8.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
+	golang.org/x/text v0.8.0 // indirect
+	google.golang.org/genproto v0.0.0-20210831024726-fe130286e0e2 // indirect
+	google.golang.org/grpc v1.40.0 // indirect
+	google.golang.org/protobuf v1.28.0 // indirect
+	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+	k8s.io/klog/v2 v2.30.0 // indirect
+	k8s.io/kubelet v0.23.0 // indirect
```

Comment:
@MarSik how much dependencies will be stripped if you copy out the private struct you need kubelet for?